game-core/src/main/java/games/strategy/triplea/attachments/UnitAttachment.java

```diff
@@ -226,6 +226,10 @@
   // Manually set TUV
   private int tuv = -1;
 
+  // combo properties
```

Comment:
A 'combo' property is not well defined, what does this mean? Is this comment adding more than the questions it raises? I would suggest either explaining this more or just omitting the comment.